# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/content-security-policy/frame-ancestors/frame-ancestors-from-serviceworker.https.html.ini
# Commit: e21ad27bfd0a
# Full Hash: e21ad27bfd0a2fef90919101eaef5aa5af1cc6c2
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-10-22 21:43:14
# Regressor Bug: 1584993
# File Overlap Count: 1
# Description:
#   Bug 1584993: Make CSP frame-ancestors work with fission enabled. r=jkt,farre,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49147
# ==============================================================================

diff -r d9d678e7422e -r e21ad27bfd0a testing/web-platform/meta/content-security-policy/frame-ancestors/frame-ancestors-from-serviceworker.https.html.ini
--- a/testing/web-platform/meta/content-security-policy/frame-ancestors/frame-ancestors-from-serviceworker.https.html.ini	Tue Oct 22 09:37:38 2019 +0000
+++ b/testing/web-platform/meta/content-security-policy/frame-ancestors/frame-ancestors-from-serviceworker.https.html.ini	Tue Oct 22 10:57:43 2019 +0000
@@ -1,5 +1,4 @@
 [frame-ancestors-from-serviceworker.https.html]
-  expected: TIMEOUT
   [A 'frame-ancestors' CSP directive set from a serviceworker response with a value 'none' should block rendering.]
-    expected: TIMEOUT
+    expected: FAIL
 
